var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
 from spack import *
```

Comment:
Apologies: our CI has been fixed, but the flake checks have been updated and now require a newline between "import groups" (e.g. Spack imports are one group, and Python core libs are another).

Reply:
Ok. NP